lte/gateway/c/session_manager/ShardTracker.h

```diff
@@ -47,7 +59,7 @@ class ShardTracker {
   /*
    * largest number of UEs that can fill a shard
    */
-  const uint16_t max_shard_size_ = 100;
+  const uint16_t max_shard_size_ = 300;
```

Comment:
Default value I have set from running the tests(on UEs at multiples of 100, 600-800 UEs). Three shards is a good number to have(though ten has performed slightly better).

Reply:
Let's put this comment inside the code so that we don't lose context :)